docs/01-start-here/06-testing-tips.md

```diff
@@ -0,0 +1,157 @@
+ It is interesting from a testing perspective because it is a different operating system, different browser setup and different screen-size from the machine you are developing on.
+ This combination often throws up interesting layout problems. In addition Safari often has less support for new features compared with Chome, which can also lead to rendering errors.
+
+- IE9 - This is the second most popular IE version. IE is notorious for having less support for features and also being much less tolerant of small errors than Chrome, or Firefox. Often when a page renders on those browsers it will still have problems on IE
+ Add to this the fact that it is much older than IE11 so supports even less features. IE9 is the oldest browser version that we expect to be fully supported. If it works on IE9 it is a good indication it will work on most other browsers.
+
+- Chrome and Firefox on windows 10 - IE10 is slowly overtaking Windows7 as our users most popular desktop OS. When Windows 10 originally came out, firefox had some rendering issues and Chrome and firefox periodically show redering issues there - especially with things like fonts.
+
+- Older devices - A common source of problems comes from older devices. Where the device is running the latest operating system, it will attempt to run all the latest features and give the fullest experience. This can be catastropihc if the device just doesnt have the memory or processing power to cope.
+  This is especially important if your feature is very resource heavy. For reasons unknown, but possibly connected to the way safari manages memory, iOS devices have proved more susceptable
+   Good examples of old devices to test on are iPhone 4S, iPad3 (both known to be prone to crashes) and the Samsung Galaxy S3.
+    [Interesting page talking about iOS crashes](http://stackoverflow.com/questions/22039534/ios-browser-crashes-due-to-low-memory)
+
+- Devices with strange screen sizes. If you re interested in layout and rendering for your change, the iphone 5S is a popular device with an abnormally narrow screen,
+    the Galaxy S4 mini is a miniaturised version of the GalaxyS4 with a very small screen. iPad Mini and Galaxy Note also have interesting dimensions and have shown layout issues from time to time
+     -especially on fronts
+
+- Rotate your device - another layout trick that is always worth doing. Load the page and rotate from portrait to landscape.
+   Refresh and rotate from landscape to portrait. This is a common cause of redrawing in the real world that can often show up issues.
+    A good device to test this on is the Nexus7 - this is because the portrait and landscape views on this device correspond to different screen-width breakpoints.
+
+- Test on AMP! [Google are making amp the default in mobile search results](http://searchengineland.com/amp-live-in-google-259109).
```

Comment:
I wonder whether amp should get it's own section lower down, I think there's other stuff after it but amp (and I will do one for ad testing as well) should be sub-sections?